tests/tango/test_signal_data_packet.py

```diff
@@ -57,7 +57,7 @@ def test_data_packet(simple_data_packet):
     assert packet.fractional_timestamp[1] == 0x10000000
     assert packet.trailer[0] == 0xAAAAAAAA
     assert packet.trailer[1] == 0xFFFFFFFF
-    assert packet.time == 1218456.504606846976
+    assert packet.time == 65535.000268435455
```

Comment:
This test change requires an explaination:
In the Mock data TSI = 0x0000FFFF, TSF0 = 0x00000000, and TSF1=0x10000000.
The equation to determine time is TSI + ((TSF0 << 32) + TSF1) *1e-12. TSF0 and TSF1 are the fractional parts of time while TSI is the integer part.
65535.000268435455 is the correct answer for time given our mock data. The previous test was written incorrectly. 1218456.504606846976 would be the correct answer is TSF1 = 0x1000000000000000. This is where I believe the problem arose.
Numpy's implicit conversion from uint32 to int64 changed the data and produced an erroneous result.